apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -56,6 +57,29 @@ export interface IEventHooksJson {
 }
 
 /**
+ * Part of IRushConfigurationJson.
+ */
+export interface IVersionPolicyJson {
+  policyName: string;
+  baseType: string;
```

Comment:
enum won't work here.

---
In reply to: [125350862](https://github.com/Microsoft/web-build-tools/pull/259#discussion_r125350862) [](ancestors = 125350862)